closed/make/Main.gmk

```diff
@@ -38,8 +38,10 @@ openssl-build :
 j9vm-build : openssl-build
 	+$(OPENJ9_MAKE) OPENJDK_VERSION_NUMBER_FOUR_POSITIONS=$(VERSION_NUMBER_FOUR_POSITIONS) build-j9
 
-j9vm-compose-buildjvm : j9vm-build
+j9vm-compose-buildjvm : images-only
```

Comment:
I think this needs to depend upon `exploded-images` and not just `images-only`, otherwise `java` may not run.

Reply:
`exploded-image` does not exist in Java 8, only in newer versions

Reply:
That's right, but my point is that this must depend on something that will guarantee that `java` is executable; `*-only` targets don't qualify. See the top-level Makefile:

```
$(info .  make <component>       # Build <component> and everything it depends on. )
$(info .  make <component>-only  # Build <component> only, without dependencies. This)
$(info .                         # is faster but can result in incorrect build results!)
```

Reply:
okay. maybe its better to move this back to main/Main.gmk under the images target? I can't do `j9vm-compose-buildjvm : images` here because it will not refer to the wrong images target here https://github.com/ibmruntimes/openj9-openjdk-jdk8/blob/openj9/closed/make/Main.gmk#L44

Reply:
You should be able to make your changes in `closed/main/Main.gmk`.